doc/running.rst

```diff
@@ -0,0 +1,25 @@
+Running MirgeCOM
+================
+
+Running with large numbers of ranks and nodes
+---------------------------------------------
+
+The PyOpenCL package used in MirgeCOM caches results of compilation on the hard
+disk in order to speed up multiple executions of the same kernel. This can lead
+to slowdowns on startup when executing on many ranks and/or nodes due to concurrent
+hard disk accesses. We have seen these slowdowns starting with 256 ranks/nodes.
```

Comment:
I would say this is good practice for any distributed-memory run.

Reply:
removed the 256 ranks phrase